layout.coffee

```diff
@@ -1195,6 +1203,23 @@ window.dr = do ->
       value = styleval[name](value) if name of styleval
       # console.log('setStyle', name, value, @el)
       el.style[name] = value
+      
```

Comment:
Can you redefine setStyle only when if capabilities.prefix.dom is 'WebKit'? It will run faster elsewhere that way...